internal/config/config.go

```diff
@@ -106,13 +108,38 @@ func (c *Config) applyDefaults() {
 func (c *Config) finalize() error {
 	var err error
 
+	if c.DBPath != "" {
+		if c.DBPath, err = resolvePath(c.DBPath); err != nil {
+			return fmt.Errorf("File does not exist > db_path: %w", err)
+		}
+	}
+
+	if c.Server.TLS.CertFile != "" {
+		if c.Server.TLS.CertFile, err = resolvePath(c.Server.TLS.CertFile); err != nil {
+			return fmt.Errorf("File does not exist > server.tls.cert_file: %w", err)
+		}
+	}
+	if c.Server.TLS.KeyFile != "" {
+		if c.Server.TLS.KeyFile, err = resolvePath(c.Server.TLS.KeyFile); err != nil {
+			return fmt.Errorf("File does not exist > server.tls.key_file: %w", err)
+		}
+	}
+
+	if len(c.OIDCProviders) > 0 && !c.AuthEnabled {
```

Comment:
im thinking if AuthEnabled is false we should respect that? I might like to confiure my oidc providers but still toggle auth off